src/services/quandl-service.js

```diff
@@ -75,9 +111,18 @@
 
     class QuandlService {
         constructor($resource) {
+            this.observable = new Observable();
             this.$resource = $resource;
         }
 
+        on() {
+            this.observable.on.apply(this.observable, arguments);
```

Comment:
Used apply so we could change the base "on" method signature without needing to change this one or any others who use it.
Same goes for the _notify